content/knowledge/guide.md

```
@@ -0,0 +1,1725 @@
+
+# 時間帯別ペイント可否表
+
+| 項目 | 午前 | 午後 | 応援ダンス | 騎馬戦 |
+|------|:---:|:---:|:---:|:---:|
+| 髪ゴム・髪単体の編み込み | ○ | ○ | ○ | ○ |
+| リボン・エクステ | × | ○ | ○ | × |
+| ラメ・シール・ストーン | × | × | × | × |
+| ペイント・髪染め | × | ○※1 | ○ | ×※2 |
+| コスプレ等の異装 | × | × | ×※3 | × |
+
+### 注釈
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)